content/docs/user-guide/how-to/reverse-common-mistakes.md

```diff
@@ -0,0 +1,45 @@
+# How to Reverse Common Mistakes
```

Comment:
hmm ... I get the idea, but feel to early for such generalization

I think even Stop tracking is better if we can't find anything else ... let's do that one

Reply:
K, done.